web/models/friend.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Entice.Web.Friend do
+  use Ecto.Model
+
+  #has_one :friend_account_id because the 
+  #friend account can be deleted before the 
+  #friend object itself
+  schema "friends" do
+    belongs_to  :account_id,            Entice.Web.Account
+    has_one     :friend_account_id,     Entice.Web.Account
```

Comment:
Cool idea :) - though I think as above we need to be careful with what foreign key we're referencing.
I'd say lets do it consistently and choose `:friend_account` here, and add the `:foreign_key` Option like so:
`has_one      :friend_account,       Entice.Web.Account,       foreign_key: :account_id`
Btw, I think the SQL table definition in the migration is correct, but I'm super unsure, so please test that stuff :D